examples/chef/esp32/CMakeLists.txt

```diff
@@ -48,6 +48,7 @@ if(NOT ${CONFIG_DEVICE_PRODUCT_NAME} STREQUAL "")
 endif()
 if(NOT ${CHIP_DEVICE_CONFIG_DEVICE_SOFTWARE_VERSION_STRING} STREQUAL "")
     idf_build_set_property(COMPILE_OPTIONS "-DCHIP_DEVICE_CONFIG_DEVICE_SOFTWARE_VERSION_STRING=\"${CHIP_DEVICE_CONFIG_DEVICE_SOFTWARE_VERSION_STRING}\"" APPEND)
+    set(PROJECT_VER ${CHIP_DEVICE_CONFIG_DEVICE_SOFTWARE_VERSION_STRING})
```

Comment:
Please add a comment that describes why this has to be done like this here: because it needed to have it follow to the software image.